### DISCUSSION TITLE
Multiple Admins - only first to access gets set

### DISCUSSION THREAD

**Discussion by dcieslak19973** [open-webui/open-webui#4996] [General]
# Bug Report

## Installation Method

Running in kubernetes

## Environment

- **Open WebUI Version: v0.3.16
- **Ollama - N/A

- **Operating System:** N/A
- **Browser (if applicable):** Chrom

**Confirmation:**

- [X] I have read and followed all the instructions provided in the README.md.
- [X] I am on the latest version of both Open WebUI and Ollama.
- [] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

I have set SHOW_ADMIN_DETAILS to True and ADMIN_EMAIL to "admin1@example.com,admin2@example.com".  I would expect when each of admin1@example.com and admin2@example.com login, they will both be set to admins.

## Actual Behavior:

Only the first to login of admin1@example.com and admin2@example.com is set to admin.  That person can then make the other an admin thru the Admin Settings, but it would be nice not to have to do this step.

## Description

**Bug Summary:**
ADMIN_EMAIL should be understood to be a comma-separated list of admin users.

## Reproduction Details

**Steps to Reproduce:**
Set SHOW_ADMIN_DETAILS to True and ADMIN_EMAIL to a list of email addresses.  Observe that only the first of these email addresses to login will initially be set to Admin.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
N/A

**Screenshots/Screen Recordings (if applicable):**
N/A

## Additional Information

N/A

**Comment by silentoplayz** [comment]
The environment variable `ADMIN_EMAIL` only accepts a single email address, which can be used to set the super-admin account's email address. The super-admin account will be the first account created for your Open WebUI instance. The `SHOW_ADMIN_DETAILS` environment variable when set to `True`, will display the super-admin account's email address on the account pending page for new account sign ups.

**Reply by dcieslak19973** [reply]
Perhaps this belongs with https://github.com/open-webui/open-webui/issues/2143

What I'd like to do is pre-define admins to avoid any manual changes on startup.  For context, we have deployed OpenWebUI in a k8s environment (currently a single pod with OIDC authentication).  As we iterate on configuration changes, we'd like to preserve user roles - whether that is through an environment variable, or membership in specific OIDC groups, either would be fine.

Separately, is there any guidance on running in a k8s environment?  How many users will a single pod support (assuming reasonable memory/CPU)?  If we run multiple pods will a user experience anything unexpected if their session routes from Pod 1 to Pod 2?
